src/types.ts

```diff
@@ -13,6 +13,7 @@ export type ImageType = {
   height: number;
 };
 export type BreedType = {
+  imgStyle?: string;
```

Comment:
I think it's better to describe imgStyle as https://www.typescriptlang.org/docs/handbook/release-notes/typescript-1-8.html#string-literal-types

define exact values of the imgStyle here, export it and use wherever you need it in the components